packages/govern-contract-utils/contracts/test/CloneFactoryMock.sol

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
I think maybe interesting to give this plugin a try: https://hardhat.org/plugins/eth-optimisim-smock.html#mockcontractfunction

I'll try to use it on my tests and let you know 👍

Reply:
Yes, this is definitely something we can use in some cases. This specific case doesn't include it :-)